src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -89,11 +101,77 @@ private void PopupWindow_Opened(object sender, EventArgs e)
         {
             if (hostControlInfo.HtmlPage != null && !string.IsNullOrEmpty(hostControlInfo.HtmlPage.FileName))
             {
-                ContentRichTextBox.Visibility = Visibility.Hidden;
+                ContentRichTextBox.Visibility = Visibility.Hidden;            
+                InitWebView2Component();
+            }
+        }
 
-                webBrowserWindow = new WebBrowserWindow(popupViewModel, hostControlInfo);
-                webBrowserWindow.IsOpen = true;
+        private async void InitWebView2Component()
+        {
+            webBrowserComponent = new WebView2();
+            webBrowserComponent.Margin = new System.Windows.Thickness(popupBordersOffSet, 0, 0, 0);
+            webBrowserComponent.Width = popupViewModel.Width;
+            //The height is subtracted by a const that sums the height of the header and footer of the popup
+            var heightBottom = bottomGrid.ActualHeight;
+            var heightTitle = titleGrid.ActualHeight;
+            //popupBordersOffSet * 2 because is one offset at the top and the other one at the bottom of the Popup
+            webBrowserComponent.Height = popupViewModel.Height - (heightBottom + heightTitle + popupBordersOffSet * 2);
+            contentGrid.Children.Add(webBrowserComponent);
+            Grid.SetRow(webBrowserComponent, 1);
+
+            LoadWebBrowser(hostControlInfo.HtmlPage);
```

Comment:
@RobertGlobant20 Can these code be reused for future tasks, e.g. replacing library view and documentation browser? Just curious

Reply:
The methods LoadWebBrowser, LoadResouces and LoadResourceAndReplaceByKey were moved to the ResourceUtilities class so they can be used in other places, please check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13142/commits/f8c0b4b52c2de77495455df98ec5bdb11fdc496f